babel/src/test/resources/sql/big-query.iq

```diff
@@ -148,23 +148,25 @@ SELECT current_date() AS the_date, t.current_date FROM t;
 #
 # Returns DATETIME
 
-!if (false) {
-SELECT CURRENT_DATETIME() as now;
-+----------------------------+
-| now                        |
-+----------------------------+
-| 2016-05-19T10:38:47.046465 |
-+----------------------------+
+SELECT CURRENT_DATETIME() > DATETIME '2008-12-25 15:30:00' as now;
++------+
+| now  |
++------+
+| true |
++------+
+(1 row)
+
 !ok
 
-SELECT CURRENT_DATETIME as now;
-+----------------------------+
-| now                        |
-+----------------------------+
-| 2016-05-19T10:38:47.046465 |
-+----------------------------+
+SELECT CURRENT_DATETIME > DATETIME '2008-12-25 15:30:00' as now;
++------+
```

Comment:
Would it be useful if Quidem had a way to mask out results? E.g.
```
!mask 6 "[0-9]" "x"
SELECT CURRENT_DATETIME as now;
+----------------------------+
| now                        |
+----------------------------+
| 2016-05-19T10:38:47.046465 |
+----------------------------+
```
would succeed because '2016-05-19T10:38:47.046465' and '2023-11-11T16:38:47.012345' would both be converted to 'xxxx-xx-xxTxx:xx:xx.xxxxxx'. The 6 means 'the next 6 lines'.

I logged https://github.com/julianhyde/quidem/issues/79. Please add comments there.

Reply:
Left a comment on the issue, sounds like a good idea.